modules/renderer/Renderer.js

```diff
@@ -114,15 +114,15 @@ export default function Renderer(config = { }) {
      * @param {FontFace} fontFace - fontFace which gets rendered
      * @return {string} fontFamily reference
      */
-    renderFont(family, files, properties = { }) {
+    renderFont(family, files, props = { }) {
```

Comment:
I actually used `properties` to show that those are **not** like the other `props` but rather a set of only 5 valid properties that can be set additionally